src/core/installAddon.js

```diff
@@ -298,6 +298,7 @@ export class WithInstallHelpers extends React.Component {
     WrappedComponent: PropTypes.func.isRequired,
     _addonManager: PropTypes.object,
     _tracking: PropTypes.object,
+    addon: PropTypes.object,
```

Comment:
We have the guid in the props already, no?

Reply:
linting was complaining about this so I added this here

Reply:
actually after changing to async/await, it seems that we can remove..